### PR TITLE
Costa Rica

### PR DESCRIPTION
**Pull request by zccaari**
# Costa Rica

Created costa_rica.md file with info

# README

Modified the README.md for South America to include an internal link to Costa Rica

Answers UCL-RITS/rse-classwork-2020#4

@Jaymee

### CODE REVIEW

File: south_america/README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # SOUTH_AMERICA
+-[COSTA_RICA](./south_america/costa_rica/costa_rica.md)
```
Comment (dpshelio) [inline south_america/README.md:2]: something wrong in this line? 😉

**Review by dpshelio — CHANGES_REQUESTED** [review]
I think the problem now is that the test is expecting a README file under each directory, so if this is a description of the country it should go under `south_america/costa_rica/README.md`

**Review by dpshelio — APPROVED** [review]
🎉

**Comment by zccaari** [comment]
Thanks for your help !

**Pull request merged**